### DISCUSSION TITLE
Installation without rpm

### DISCUSSION THREAD

**Discussion by gapfranco** [saadeghi/daisyui#2618] [Q&A]
I installed Tailwind with its standalone cli and don't use rpm. Can I install daisyui without using rpm too?

**Comment by saadeghi** [comment]
daisyUI is a Tailwind CSS plugin but currently Tailwind CSS standalone package only supports first-party plugins (like typography plugin) and not third parties.  

You can use daisyUI CDN file:  
https://daisyui.com/docs/cdn/

And if there's a non-js CSS purging tools that can remove unused CSS that would be great. Otherwise the downside of using CDN file would be unused styles in production.

Let me know if you have any questions

**Reply by gapfranco** [reply]
Thank you for the reply. I just don't want to install node or npm in my machine as I would not use it for anything else and I find them too cumbersome.
